tests/test_load_lib.py

```diff
@@ -34,16 +35,29 @@
 */
 """
 
+cpp_code_no_pybind = """// cppimport
+
+int square(int x) {
```

Comment:
?

Reply:
One code snippet includes the pybind code while one does not. You need pybind depending on how you load the module

```PYBIND11_MODULE(module, m) {
    m.def("square", &square);
}```